samples/bluetooth/mesh/light/boards/nrf5340dk_nrf5340_cpuapp_ns.conf

```diff
@@ -12,3 +12,26 @@ CONFIG_BT_HOST_CRYPTO=n
 CONFIG_BT_GATT_CACHING=n
 
 CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=n
+
+# Enable point to point DFU over SMP
+CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU=y
+CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP=y
+
+# Enable updates for the network core
+CONFIG_NRF53_UPGRADE_NETWORK_CORE=y
+CONFIG_UPDATEABLE_IMAGE_NUMBER=2
```

Comment:
Why is updatable image number 2? It should be at least 3 (secure image + non-secure image + network core). Did I miss anything?